### ISSUE TITLE
does not tell me how SM-C is licensed

### ISSUE THREAD

**Issue by merdos** [NetAppDocs/ontap#644]
Page: [SnapMirror licensing overview](https://docs.netapp.com/us-en/ontap/data-protection/snapmirror-licensing-concept.html)

talks about a lot of things, but not about how it is licensed. I believe capacity based and term based.

**Comment by netapp-forry** [comment]
Thanks for letting us know, Mark -- we'll look into this ASAP.

**Comment by netapp-lenida** [comment]
I've updated the SnapMirror Cloud license section of this topic after working with the product manager on some additional content. The updated text includes information about how SnapMirror Cloud is licensed:

"SnapMirror Cloud is not available as a standalone license and is available only with purchase of the Hybrid Cloud Bundle. Hybrid Cloud Bundle is a term-based subscription license that is priced based on capacity. Only one license is needed per ONTAP cluster. Capacity is defined as the “used” capacity (not raw capacity) within any volume which is protected by SnapMirror Cloud. Users will purchase this license based on the total used capacity of volumes on the cluster being backed up by SnapMirror Cloud. As of October 2021, the Hybrid Cloud Bundle includes only a SnapMirror Cloud license (previously Hybrid Cloud Bundle included a FabricPool license, which was removed from the bundle effective October 2021). In addition to SnapMirror Cloud, the async SnapMirror license is also required and is available only with the purchase of the Data Protection Bundle." 

Thanks again for contact us.
